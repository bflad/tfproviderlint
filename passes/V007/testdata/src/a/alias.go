package a

import (
	v "github.com/hashicorp/terraform-plugin-sdk/helper/validation"
)

func falias() {
	// Passing

	_ = v.StringIsValidRegExp

	// Comment ignored

	//lintignore:V007
	_ = v.ValidateRegexp

	_ = v.ValidateRegexp //lintignore:V007

	// Failing

	_ = v.ValidateRegexp // want "deprecated v.ValidateRegexp should be replaced with v.StringIsValidRegExp"
}
