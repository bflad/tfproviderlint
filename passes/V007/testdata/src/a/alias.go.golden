package a

import (
	v "github.com/hashicorp/terraform-plugin-sdk/helper/validation"
)

func falias() {
	// Passing

	_ = v.StringIsValidRegExp

	// Comment ignored

	//lintignore:V007
	_ = v.ValidateRegexp

	_ = v.ValidateRegexp //lintignore:V007

	// Failing

	_ = v.StringIsValidRegExp // want "deprecated v.ValidateRegexp should be replaced with v.StringIsValidRegExp"
}
